gossip/src/cluster_info.rs

```diff
@@ -2160,27 +2161,21 @@ impl ClusterInfo {
         I: IntoIterator<Item = (SocketAddr, Ping)>,
     {
         let keypair = self.keypair();
-        let packets: Vec<_> = pings
+        let pongs_and_dests: Vec<_> = pings
             .into_iter()
             .filter_map(|(addr, ping)| {
                 let pong = Pong::new(&ping, &keypair).ok()?;
                 let pong = Protocol::PongMessage(pong);
-                match Packet::from_data(Some(&addr), pong) {
-                    Ok(packet) => Some(packet),
-                    Err(err) => {
-                        error!("failed to write pong packet: {:?}", err);
-                        None
```

Comment:
Technically this is a slight change of behavior; `Packet::form_data()` can fail if the enclosed call to `bincode::serialize_into()` fails.
- Old behavior would exclude any pongs that couldn't be serialized with via the surrounding `filter_map()` before going into the batch
- New behavior would have a spot for the packet in the batch, but he packet wouldn't be properly initialized (ie blank address)

However, I think this is not a concern given that the `Protocol` fits in the packet. Additionally, this "new" behavior is what the other `ClusterInfo` functions that are updated to use newly named function have